python/lsst/afw/geom/linearTransform.cc

```diff
@@ -59,37 +62,50 @@ PYBIND11_PLUGIN(_linearTransform) {
     clsLinearTransform.def(py::init<typename LinearTransform::Matrix const &>());
 
     /* Operators */
-    clsLinearTransform.def("__getitem__", [](LinearTransform const & self, int i) {
-            return self[lsst::utils::cppIndex(4, i)];
-        }, py::is_operator());
-    clsLinearTransform.def("__getitem__", [](LinearTransform const & self, std::pair<int, int> i) {
-            auto row = lsst::utils::cppIndex(2, i.first);
-            auto col = lsst::utils::cppIndex(2, i.second);
-            return self.getMatrix()(row, col);
-        }, py::is_operator());
-
+    clsLinearTransform.def("__call__",
+                           (Point2D(LinearTransform::*)(Point2D const &) const) & LinearTransform::operator(),
+                           py::is_operator());
+    clsLinearTransform.def(
+        "__call__", (Extent2D(LinearTransform::*)(Extent2D const &) const) & LinearTransform::operator(),
+        py::is_operator());
+    clsLinearTransform.def(
+        "__getitem__", [](LinearTransform const &self, int i) { return self[lsst::utils::cppIndex(4, i)]; },
+        py::is_operator());
+    clsLinearTransform.def("__getitem__",
+                           [](LinearTransform const &self, std::pair<int, int> i) {
+                               auto row = lsst::utils::cppIndex(2, i.first);
+                               auto col = lsst::utils::cppIndex(2, i.second);
```

Comment:
I don't really like the magic numbers for the size of the collection. Not sure if it can be avoided either. But perhaps a simple `constexpr std::ptrdiff_t size = 2;` will clarify the intent enough.